src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.ts

```diff
@@ -109,6 +121,10 @@ export class CustomerListComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   deleteCustomer() {
+    if (this.idToDelete === this.idToEdit) {
```

Comment:
I think it is necessary to check these conditions in the following cases:
- When select te project **A** to edit and clicked in other project **B** options Delete 🗑.
- when select  the project  **A** with the option edit and click in delete 🗑.

Reply:
- In the first case the data of project **A** is kept in the form and project **b** is deleted. 
- In the second case the form is reset and project **A** is deleted.